src/components/PracticeSection/MultipleChoice.js

```diff
@@ -251,11 +256,11 @@ export default function MultipleChoice() {
           const reward = getExplanationReward(totalexplanation);
           student.add("reward_badge_ids", reward);
           const points = student.get("total_points");
-          const rewardPoints = points + 50;
+          //const rewardPoints = points + 50;
```

Comment:
I can see now that there are more comments which should be removed :)

Reply:
Oki. Please remove them and explicitly tag me when this is ready to merge? Thanks!